src/DotNetProjectFile.Analyzers/Rule.cs

```diff
@@ -513,6 +513,17 @@ public static class Rule
        category: Category.Bug,
        isEnabled: true);
 
+    public static DiagnosticDescriptor SomeRule => New(
+       id: 2100,
+       title: "Add invariant fallback values",
+       message: "Mark this <PackageReference> as a private asset.",
```

Comment:
Perhaps something like "Dependencies only used during compilation time should be marked as private assets."